compute/managed-instances/demo/main.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python3
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+from os import path
+from requests import get
+from re import sub
+from socket import gethostname
+from socketserver import ThreadingMixIn
+
+PORT_NUMBER = 80
+HTML_TEMPLATES_PATH = path.dirname(path.realpath(__file__)) + '/templates'
+HEALTHY = True
```

Comment:
Does this sample ever have multiple instances behind a load balancer, or is it intended to run on one machine at a time only?

Reply:
In the future, I intend to run this sample with a load balancer.

The global 'healthy' is meant to simply hold state for each individual server.  It is expected that a user might be redirected to different servers by the load balancer.  Please let me know if there is a more idiomatic way of hosting a simple boolean state variable on a flask server (all I can find are examples using databases which seems too overcomplicated for this task).